teaser/data/input/usecond_input.py

```diff
@@ -33,33 +33,37 @@ def load_use_conditions(use_cond, zone_usage, data_class):
     use_cond.with_cooling = conditions_bind[zone_usage]["with_cooling"]
     use_cond.persons = conditions_bind[zone_usage]["persons"]
     use_cond.ratio_conv_rad_persons = conditions_bind[zone_usage][
-        "ratio_conv_rad_persons"]
+        "ratio_conv_rad_persons"
+    ]
     use_cond.machines = conditions_bind[zone_usage]["machines"]
     use_cond.ratio_conv_rad_machines = conditions_bind[zone_usage][
-        "ratio_conv_rad_machines"]
-    use_cond.lighting_power = conditions_bind[
-        zone_usage][
-        "lighting_power"]
-    use_cond.ratio_conv_rad_lighting = conditions_bind[
-        zone_usage]["ratio_conv_rad_lighting"]
-    use_cond.use_constant_infiltration = conditions_bind[
-        zone_usage]["use_constant_infiltration"]
-    use_cond.infiltration_rate = conditions_bind[zone_usage][
-        "infiltration_rate"]
-    use_cond.max_user_infiltration = conditions_bind[
-        zone_usage]["max_user_infiltration"]
-    use_cond.max_overheating_infiltration = conditions_bind[
-        zone_usage]["max_overheating_infiltration"]
-    use_cond.max_summer_infiltration = conditions_bind[
-        zone_usage]["max_summer_infiltration"]
-    use_cond.winter_reduction_infiltration = conditions_bind[
-        zone_usage]["winter_reduction_infiltration"]
+        "ratio_conv_rad_machines"
+    ]
```

Comment:
are these line breaks on purpose?

Reply:
this is automatically done by python-black, so yeah wuite on purpose